.github/workflows/integration_test.yml

```diff
@@ -5,10 +5,11 @@ on: [pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
What does this do?

Reply:
We run integration tests for 2 different Python version (`matrix` mode). When one of them fail, by default the other one stops as well. 
I disabled this behavior so they all run until the end when I was doing some testing. I think it's still a good thing to let them all run in case there's a difference in behavior between Python version that we don't catch since tests end early
Documentation: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures